modules/calib3d/src/ptsetreg.cpp

```diff
@@ -207,24 +206,21 @@ class RANSACPointSetRegistrator : public PointSetRegistrator
                 }
             }
 
-            nmodels = cb->runKernel( ms1, ms2, model );
-            if( nmodels <= 0 )
+            int success = cb->runKernel( ms1, ms2, model );
```

Comment:
`cb->runKernel` simply returns if `solvePnP` succeeded or not:
https://github.com/opencv/opencv/blob/fcec053d59a8e30cbac7db571fcd448bfc98dd53/modules/calib3d/src/solvepnp.cpp#L215